### PR TITLE
Surface groups, remotes, distribution permissions on /me endpoint

### PR DESCRIPTION
**Pull request by rochacbruno**
Issue: #468

Example output:

```diff
galaxy_ng on  468 [$!+] via  v3.8.5 (galaxy_ng) took 28m42s
❯ galaxy /api/automation-hub/_ui/v1/me/
HTTP/1.1 200 OK
X-Powered-By: Express
allow: GET, PUT, HEAD, OPTIONS
connection: close
content-length: 607
content-type: application/json
date: Fri, 02 Oct 2020 20:59:12 GMT
server: gunicorn/20.0.4
vary: Accept, Cookie
x-frame-options: SAMEORIGIN

{
    "date_joined": "2020-01-01T00:00:00Z",
    "email": "admin@example.com",
    "first_name": "",
    "groups": [
        {
            "id": 1,
            "name": "system:partner-engineers"
        }
    ],
    "id": 1,
    "is_partner_engineer": true,
    "is_superuser": true,
    "last_name": "",
    "model_permissions": {
+       "add_distribution": true,
+       "add_group": true,
        "add_namespace": true,
+      "add_remote": true,
        "add_user": true,
+      "change_distribution": true,
        "change_namespace": true,
        "change_remote": true,
        "change_user": true,
+      "delete_distribution": true,
+      "delete_group": true,
+      "delete_remote": true,
        "delete_user": true,
        "move_collection": true,
+      "update_group": true,
        "upload_to_namespace": true,
        "view_user": true
    },
    "username": "admin"
}

```

### CODE REVIEW

File: galaxy_ng/app/api/ui/serializers/user.py [hunk 1]
```diff
@@ -115,8 +115,16 @@ def get_model_permissions(self, obj):
...
+            "change_distribution": obj.has_perm('ansible.change_ansibledistribution'),
+            "delete_distribution": obj.has_perm('ansible.delete_ansibledistribution'),
+            "add_distribution": obj.has_perm('ansible.add_ansibledistribution'),
             "view_user": obj.has_perm('galaxy.view_user'),
             "delete_user": obj.has_perm('galaxy.delete_user'),
             "change_user": obj.has_perm('galaxy.change_user'),
             "add_user": obj.has_perm('galaxy.add_user'),
+            "add_group": obj.has_perm('galaxy.add_group'),
+            "delete_group": obj.has_perm('galaxy.delete_group'),
+            "update_group": obj.has_perm('galaxy.update_group'),
```
Comment (newswangerd) [inline galaxy_ng/app/api/ui/serializers/user.py:129]: should be `change_group`

**Comment by ZitaNemeckova** [comment]
@newswangerd @rochacbruno What about `view_group` and `view_distribution` permissions? Sounds like what I need for https://github.com/ansible/galaxy_ng/issues/467

**Review by newswangerd — APPROVED** [review]

**Comment by newswangerd** [comment]
> @newswangerd @rochacbruno What about `view_group` and `view_distribution` permissions? Sounds like what I need for #467

Oh yeah, you should add these. I forgot about them

**Comment by rochacbruno** [comment]
@ZitaNemeckova @newswangerd  added view_group and view_distribution

**Pull request merged**